Hack in support for two OCR attempts at candy name

lramati opened the pull request:
Currently the calibrator only sets up one field, which the scan logic then splits into two. This split should probably occur in the calibrator, to open the possibility to configure each separately

Also need to add code for handling new offsets (see Issue #10) for more details before this can really be considered done

lramati committed:
Hack in support for two OCR attempts at candy name

Sihoiba commented on app/src/main/java/com/kamron/pogoiv/Pokefly.java:
```diff
@@ -204,7 +206,7 @@ public static Intent createNoInfoIntent() {
         intent.putExtra(KEY_SEND_INFO_NAME, scanData.getPokemonName());
         intent.putExtra(KEY_SEND_INFO_TYPE, scanData.getPokemonType());
-        intent.putExtra(KEY_SEND_INFO_CANDY, scanData.getCandyName());
+        intent.putExtra(KEY_SEND_INFO_CANDY, scanData.getCandyNames().toArray());
```
Can getCandyNames return null?

lramati replied:
Under current logic flows, no, but I'll add some protection when initializing a new ScanData just in case

lramati committed:
Some Null saftey

lramati committed:
Disable pokespam because its super broken

lramati committed:
Add logic for candy offsets to field scanner

lramati committed:
handle edge case when scanning bad fields
old implementation of fixOcrLettersToNums could cause logic to think
it found a good value when reading garbage, which would break scanning.
New version is more robust (but likely not perfect)

lramati committed:
bump version

lramati committed:
Workaround to get default form on multiform pokes
default "" input handled differently, will never return null

lramati committed:
Change strings files to recommend Caterpie for recalibration

lramati committed:
Bump targetSdk version because appstore says so

lramati committed:
Attempts were made to get targetsdk=29 to work

Commit:
Change output directory for APKs.

lramati committed:
add .aab bundle files to gitignore

lramati committed:
API version 29 compliance part 1

lramati committed:
Don't "fix" deprecated code without testing

lramati committed:
Split pokefly launch across two intents
This solves the timing issue with creating ScreenGrabber

lramati committed:
Merge pull request #1 from jimmywan/jw.change_output_dir

jw.change_output_dir

lramati committed:
Bump version to 5.5.2

lramati committed:
Revert "jw.change_output_dir"

lramati committed:
Merge pull request #2 from lramati/revert-1-jw.change_output_dir

Revert "jw.change_output_dir"

lramati committed:
Launch Pokemon Go in screenshot mode too

lramati committed:
service.startForeground has different args in v29+

lramati committed:
Fix too tall candy name area calibration error

lramati committed:
Allow calibration on pokemon with wrapped candy text

lramati committed:
bump version (including calibration version)

lramati committed:
Move final scan adjustments to after displaying results

lramati committed:
Update recalibration warning

lramati committed:
Update recalibration warning

lramati committed:
Bump version num to full release
